lib/GroupHelper.php

```diff
@@ -61,6 +61,10 @@ public function addEvent(int $id, IEvent $event): void {
 				} else {
 					$event = $provider->parse($language, $event);
 				}
+
+				if ($event->isValidParsed()) {
+					$this->logger->info('Activity event was claimed to be parsed, but was not fully parsed by ' . get_class($provider) . ' [app: ' . $event->getApp() . ', subject: ' . $event->getSubject() . ']');
```

Comment:
consider

```suggestion
					$this->logger->info('Activity event was claimed to be parsed, but was not fully parsed by ' . get_class($provider) . ' [app: ' . $event->getApp() . ', subject: ' . $event->getSubject() . ']', ['app' => $event->getApp()]);
```

so logs and Sentry point to the problematic app and not the activity app. same below.

Reply:
Makes sense, done